README.md

```diff
@@ -41,11 +41,134 @@ Running your app using IPFS will mimic the production environment that will be u
 - **build:app**: Builds the front-end
 - **build:script**: Builds the background script
 - **test**: Runs tests for the contracts
-- **publish:minor**: Release a minor version to aragonPM
+- **publish:patch**: Release a patch version of the frontend to aragonPM
+- **publish:minor**: Release a minor version of the frontend to aragonPM
 - **publish:major**: Release a major version to aragonPM with a potentially new contract address for on-chain upgrades
+- **versions**: Check the currently installed versions of the app
 
 ### Libraries
 
 - [**@aragon/os**](https://github.com/aragon/aragonos): Aragon interfaces
 - [**@aragon/client**](https://github.com/aragon/aragon.js/tree/master/packages/aragon-client): Wrapper for Aragon application RPC
 - [**@aragon/ui**](https://github.com/aragon/aragon-ui): Aragon UI components (in React)
+
+## Publish
+This app has 3 environments defined in `arapp.json`:
+
+| Environment   | Network   |
+|---            |---        |
+| default       | localhost |
+| staging       | rinkeby   |
+| production    | mainnet   |
+
+Prerequisites:
+- ENS Registry address
+
+Note: the `default` environment which points to `localhost` does not have an ENS Registry address specified, that's because the `@aragon/cli` will use the default value of `0xB9462EF3441346dBc6E49236Edbb0dF207db09B7`.
```

Comment:
Would reword as:

```
... because the `@aragon/cli` will default the value to `0xB9462EF3441346dBc6E49236Edbb0dF207db09B7` (the ENS Registry pre-deployed on the local development chain).
```